CHANGES.md

```diff
@@ -2,6 +2,7 @@
 
 #### next release (8.2.24)
 
+- Fix `Key Share URL use case broken: share URL for Compare datasets forgets 2nd date`
```

Comment:
It's better to note the bug that was fixed rather than a specific case of it (although it can be good list some specific use cases in a sub-list under the main bug fix entry). Sorry, I don't have a suggestion of a better changelog entry until I've understood the code change in SplitItemReference.ts.